### ISSUE TITLE
Default cert is not presented when LB is accessed with certs other than supported certs/default_cert.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#8365]
Rancher Versions: Build from master
Haproxy image - prachidamle/lb-service-haproxy:latest

Created LB with the following cert directories for cert and default_cert , where  /certs/mycerts directory had certs for test1.com
```
      io.rancher.lb_service.cert_dir: /certs/mycerts
      io.rancher.lb_service.default_cert_dir: /certs/test5.com
```

When accessing this LB using certs for "test2.com" (anything other than test1.com/test5.com), user is presented with test1.com certs. 
```
root@san-nfs1:~/testlb# wget --header="host: test2.com" --ca-certificate=/root/certs/mycerts/mycerts/certs/test2.com/test2.com.crt https://test2.com:8086/name.html
--2017-03-31 01:03:15--  https://test2.com:8086/name.html
Resolving test2.com (test2.com)... *****
Connecting to test2.com (test2.com)|*****|:8086... connected.
ERROR: cannot verify test2.com's certificate, issued by ‘/C=AU/ST=Some-State/O=Internet Widgits Pty Ltd/OU=test1.com/CN=test1.com’:
  Self-signed certificate encountered.
    ERROR: certificate common name ‘test1.com’ doesn't match requested host name ‘test2.com’.
To connect to test2.com insecurely, use `--no-check-certificate'.
root@san-nfs1:~/testlb# 
```
Expected Behavior:
When accessing this LB using certs for "test2.com" (anything other than test1.com/test5.com), user should be  presented with test5.com cert which is set as the default cert.

**Comment by sangeethah** [comment]
Tested with lb image - prachidamle/lb-service-haproxy:v0.1
Tested the same case mentioned in the bug. 
When accessing this LB using certs for "test2.com" (anything other than test1.com/test5.com), user is presented with test5.com cert which is set as the default cert as expected.

**Comment by deniseschannon** [comment]
Official image released with rancher/lb-service-haproxy:v0.7.0 with the same fix.
